Source/UI/Miscellaneous/IPFSettingsViewController.swift

```diff
@@ -0,0 +1,33 @@
+import UIKit
+
+final class IPFSettingsViewController: FormViewController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+    }
+
+    private func setupUI() {
+        title = "Settings.title".localized
+
+        if #available(iOS 13.0, *) {
+            view.backgroundColor = .systemBackground
+        } else {
+            view.backgroundColor = .white
+        }
+
+        form
```

Comment:
Form and FormViewController are from a very light weight forms approach I created for building the UI). Its added under the UI/Support section of the project

Reply:
If these things are only used once I would prefer not to pull in extra code, and the style might look esoteric to other contributors, so I would stick to vanilla UI code here: can we just instantiate `SwitchRow` and `Section` instances, configure them and add them to the UI tree?

Reply:
I switched the API to be non esoteric. Rows are added now via a standard function call